site/pages/about-us/principles-slice/principle/index.js

```diff
@@ -14,9 +15,9 @@ const Principle = ({ number, title, description }) => (
 );
 
 Principle.propTypes = {
-  number: React.PropTypes.string,
-  title: React.PropTypes.string,
-  description: React.PropTypes.string,
+  number: PropTypes.string,
+  title: PropTypes.string,
+  description: PropTypes.string,
```

Comment:
This propType should be migrated to a flow type (and an `// @flow` declaration should be added on top of the file